Source/ThirdParty/embree/include/embree3/rtcore_config.h

```diff
@@ -25,13 +25,13 @@
 #define RTC_MAX_INSTANCE_LEVEL_COUNT 1
 
 #define EMBREE_STATIC_LIB
-/* #undef EMBREE_API_NAMESPACE */
+#define EMBREE_API_NAMESPACE
 
 #if defined(EMBREE_API_NAMESPACE)
-#  define RTC_NAMESPACE 
-#  define RTC_NAMESPACE_BEGIN namespace  {
+#  define RTC_NAMESPACE embree3
```

Comment:
Shouldnt `EMBREE_API_NAMESPACE` be used somehow, instead of hard-coding `embree3` here?

Reply:
This file is generated this way. I don't like it, but it's how embree is.

Reply:
If its generated it should be in `${CMAKE_BINARY_DIR}` :thinking:

Reply:
I actually don't know why `rtcore_config.h` exists in source tree in first place. Probably I copied it by accident or because it didn't work. Anyway, we are not going to change this file anytime soon, so why bother.
Maybe can fix it if/when we update embree.

Reply:
Agreed. Not a big deal. We can think about it if build system starts changing this file too often.